### ISSUE TITLE
Fluent Settings page (instead of settings.json and defaults.json)

### ISSUE THREAD

**Issue by SpiderFlashMello61319** [microsoft/terminal#5613]
# Description of the new feature/enhancement  
As the title says, it would look better and be easier to use for both programmers (who are familiar with JSON) and non-programmers. It would also solve many problems.  
  
For example, I have 6 different profiles (3 default and 3 installed), and I want to style all of them at the same time and/or in the same way. Now, the problem is I have to do the same thing 6 times (not all styles are applied by using a common color scheme). But instead, if we have a settings page which lets us choose styles for individual profiles **AND/OR** lets us make a standard master style template for all profiles, it would make styling easier and much organized.  
  
Also, everyone loves Windows 10's **Fluent Design System**, and therefore, a **Fluent Settings** page would make changing settings look much easier and more modern, and in the background, this also easily gets the work done.
  
  
  

# Proposed technical implementation details  
#### At the same time, these JSON files may not be removed.  
  
We can create a **Settings** page (which opens as another tab **OR** as a completely different section in the same window), with options as checkboxes, color-pickers, float value text boxes (for floating points like `"acrylicOpacity"`, `"padding"`, etc.), etc., and the respective settings will be applied (after clicking on an "Apply" button at the end of the settings page **OR** dynamically applying them as soon as a setting is changed) by changing the `settings.json` file (which, if wished, can be accessed by the user by opening the Windows Terminal's directory).  
  
  
# TL;DR  
We create a **Fluent Settings** interface with `AcrylicBrush` and `RevealBrush`, etc., which in the background dynamically modifies the `settings.json` file, as it fixes many boring problems (one of which has been mentioned before as an example), gives a good look and impression on Windows Terminal to many users and will be neatly welcomed.

**Comment by j4james** [comment]
See issue #1564.

**Comment by SpiderFlashMello61319** [comment]
@j4james Oh I did not know that existed. And btw, I was kinda talking about a slightly different UI than #1564   
Should I close this issue in that case?

**Comment by j4james** [comment]
Yeah, if you have suggesting for how the UI should work, it's best to add them to the original issue. You'll see there are already a bunch of comments there from different people with their own ideas. As for closing the issue, someone from MS will probably close it with a dup link eventually, but you can also close it yourself if you want.

**Comment by zadjii-msft** [comment]
Yep, this is definitely a dupe. We should continue the discussion over on that thread. Thanks!

/dup #1564
